libraries/helpers.rb

```diff
@@ -219,9 +219,9 @@ def mariadbbackup_pkg_name
       end
     end
 
-    # determine the platform specific server package name
-    def server_pkg_name
-      platform_family?('debian') ? "mariadb-server-#{new_resource.version}" : 'MariaDB-server'
+    # determine the platform specific package prefix
+    def pkg_prefix
+      platform_family?('debian') ? 'mariadb' : 'MariaDB'
```

Comment:
Instead of doing this, I think this might be a better approach since this will break anything older than 10.11:
```ruby
  def server_pkg_name
    if platform_family?('debian')
      if Gem::Version.new(new_resource.version) >= Gem::Version.new('10.11')
        'mariadb'
      else
        "mariadb-server-#{new_resource.version}"
      end
    else
      'MariaDB-server'
    end
  end
```

Reply:
The package name without the suffix still works with older versions. Installing this way also adequately addresses the problem described in #355